specs/sticky.spec.js

```diff
@@ -42,19 +43,18 @@ describe('Sticky', () => {
       }
     `});
     await page.addScriptTag({ path: 'lib/index.js' });
-    await page.evaluate(selector => {
+    await page.evaluate((selector) => {
       const { Sticky } = StuckJs;
       const element = document.querySelector(selector);
       const sticky = new Sticky(element);
     }, target);
-  }, 10000);
+  });
```

Comment:
[ASK] why did you remove `Timeout` for `beforeEach` with async function

Reply:
it's my mistake